vuework04/src/views/CheckWalletModal.vue

```diff
@@ -0,0 +1,73 @@
+  <template >
+  <transition name="modal" appear >
+    <div class="overlay" @click.self="$emit('close')" >
+      <div class="content" v-for="user of someUsers" :key="user.id">
+        {{ user.name }}さんの残高: {{ user.wallet }}
```

Comment:
これだと、someUsersのすべてのuserの情報が表示されてしまいます。
クリックされたuserの情報だけ表示させたいので、ここを変更する必要がありますね。
```suggestion
      <div class="content">
        {{ someUsers[index].name }}さんの残高: {{ someUsers[index].wallet }}
```